k8s/mongo.yml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment

```

Comment:
Creating a deployment for a stateful application?

Reply:
Added application manifests and corrected to statefulSet as needed.